updating payment page for cloud (#7145)
umbraco/UmbracoDocs: umbraco-cloud/set-up/new-shop/README.md

<figure><img src="../../.gitbook/assets/image (37).png" alt="Select Payment Methods"><figcaption><p>Select Payment Methods</p></figcaption></figure>

## Changing and removing payment methods

In some cases, you might need to change the credit card information on the Umbraco Cloud Organization.

> If you add more than one card, you can choose which one is billed on the 1st of the month. To do so click on the Set as primary button, to select that card to be billed.

Reviewer: ```suggestion
If you add multiple cards, you can choose which one will be billed on the 1st of the month. To do this, click the **Set as primary** button on the card you want to use.
```

> If you want to remove an expired card, or if you have access to a card. Click on the trashcan icon on the card to remove it.

Reviewer: ```suggestion
To remove an expired or unused card, click the trash can icon next to that card.
```
